### DISCUSSION TITLE
How to hide DOS console when start program?

### DISCUSSION THREAD

**Discussion by pankration** [hajimehoshi/ebiten#3088] [Q&A] [answered]
It shows a DOS console and disappears soon when double click .exe file, how do I do to hide this DOS console? Thx!

**Comment by hajimehoshi** [comment, marked as answer]
When you `go build`, use `-ldflags "-Hwindowsgui"`.

**Comment by pankration** [comment]
Really thanks! I also find the answer in FAQ page. [https://ebitengine.org/en/documents/faq.html#My_app_built_with_Ebitengine_creates_an_unknown_process_%22Console_Window_Host%22._What_is_this?](url)
